<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolDatabase</class>
 <widget class="QDialog" name="ToolDatabase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tool Database[*]</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="ToolEditForm" name="toolEdit" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="ToolTreeView" name="treeView"/>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbNew">
       <property name="text">
        <string>New Tool</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbNewGroup">
       <property name="text">
        <string>New Group</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCopy">
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbDelete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ToolEditForm</class>
   <extends>QWidget</extends>
   <header>tool_editform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ToolTreeView</class>
   <extends>QTreeView</extends>
   <header>tool_treeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
